MonkeyWrench.Database/Extensions/DBLogin_Extensions.cs

```diff
@@ -69,33 +71,41 @@ public static DBLogin Login (DB db, string login, string password, string ip4, b
 			return result;
 		}
 
-		public static void LoginOpenId (DB db, LoginResponse response, string email, string ip4)
+		public static void Login (DB db, LoginResponse response, string email, string ip4, List<string> userOrgs, bool useGitHub = false, string gitHubLogin = "")
 		{
-			if (string.IsNullOrEmpty (Configuration.OpenIdProvider) && string.IsNullOrEmpty (Configuration.OauthClientId))
-				throw new Exception ("No OpenId provider available");
+			string [] specs;
 
-			if (string.IsNullOrEmpty (Configuration.OpenIdRoles))
-				throw new Exception ("No OpenId roles specified");
+			string username = useGitHub ? gitHubLogin : email;
```

Comment:
Fixed in https://github.com/mono/monkeywrench/pull/74/commits/7121a27d784dc681fae54903a1ae59c42eb73964